ProcessHacker/procprv.c

```diff
@@ -1471,9 +1471,22 @@ VOID PhpFillProcessItem(
                 ProcessItem->Protection.Level = protection.Level;
             }
         }
+        else
+        {
+            // "emulate" PS_PROTECTION info for older OSes
+            PROCESS_EXTENDED_BASIC_INFORMATION basicInfo;
+
+            if (NT_SUCCESS(PhGetProcessExtendedBasicInformation(ProcessItem->QueryHandle, &basicInfo)))
```

Comment:
I've made some changes to the process provider. Remove the call to PhGetProcessExtendedBasicInformation and replace it with ProcessItem->IsProtectedProcess

Reply:
@dmex 
Thanks. Fixed.